cockatrice/src/client/tabs/tab_deck_editor.cpp

```diff
@@ -783,7 +783,7 @@ void TabDeckEditor::actNewDeck()
 
 void TabDeckEditor::actLoadDeck()
 {
-    bool openInNewTab = SettingsCache::instance().getOpenDeckInNewTab();
+    bool openInNewTab = SettingsCache::instance().getOpenDeckInNewTab() && !isBlankNewDeck();
```

Comment:
this new function is very similar to the confirmClose function, if it is true then confirmClose will also quickly return so it wouldn't be necessary to call both

Reply:
`confirmClose` only checks for modified and can pop up a window. It's also already used in a few places, so I don't want to touch it right now.

It's probably fine to have a slight bit of redundant checking. I feel like it makes the logic flow better, and the checks aren't expensive anyways.